.github/workflows/lte-integ-test-magma-deb.yml

```diff
@@ -77,3 +77,25 @@ jobs:
           check_name: LTE Debian integration test results
           junit_files: lte/gateway/test-results/**/*.xml
           check_run_annotations: all tests
+      - name: Get test logs
+        if: always()
+        run: |
+          cd lte/gateway
+          fab get_test_logs:dst_path=./logs.tar.gz
+      - name: Upload test logs
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # pin@v3
+        if: always()
+        with:
+          name: test-logs
+          path: lte/gateway/logs.tar.gz
+      - name: Publish results to Firebase
```

Comment:
Would it make sense to also add a slack notification on failure?

Reply:
Yes, sure. I added a step similar to one in the LTE integ test.